static/tutorial/fr/welcome.md

```diff
@@ -23,61 +23,64 @@ Mais, maintenant : Allons-y !
 
 Passons rapidement en revue les aspects les plus importants :
 
 1. Vous pouvez mettre le texte en **gras** et _italique_ en l'entourant de traits de soulignement ou d'astérisques. Le choix de l'astérisque ou du soulignement dépend entièrement de vous. Un seul caractère rend le texte en italique, deux le rendent gras et — vous l'avez deviné — trois le rendent à la fois ***gras et italique*** !
-2. Les titres sont créés à la manière des hashtags —  il suffit d'écrire un croisillon « # » suivi d'une espace. Vous pouvez utiliser jusqu'à six croisillons « ### » pour créer des titres de niveau 1 à 6.
-3. Les listes sont littéralement composées — il suffit d'écrire « * », « - » ou « + » sur une nouvelle ligne. Les listes numérotées sont constituées d'un nombre suivi d'un point.
-4. Enfin, les blocs de citations sont écrits exactement comme on affiche le texte cité dans les courriels : il suffit de les distinguer en utilisant « > » !
+2. Les titres sont créés à la manière des hashtags —  il suffit d'écrire un croisillon `#` suivi d'une espace. Vous pouvez utiliser jusqu'à six croisillons `######` pour créer des titres de niveau 1 à 6.
+3. Les listes sont littéralement composées — il suffit d'écrire `*`, `-` ou `+` sur une nouvelle ligne. Les listes numérotées sont constituées d'un nombre suivi d'un point.
+4. Enfin, les blocs de citations sont écrits exactement comme on affiche le texte cité dans les courriels : il suffit de les distinguer en utilisant `>` !
 
 Bien sûr, il y a beaucoup d'autres éléments. Les notes de bas de page par exemple — survolez celui-ci avec votre curseur[^1]. Au cours de ce tutoriel, vous apprendrez également certains éléments spéciaux que Zettlr utilise pour permettre un travail véritablement académique, ainsi que la gestion des connaissances à l'aide des Zettelkästen !
 
 ## Liens ⛓
 
-Bien qu'ils ne soient pas très souvent utilisés dans les textes académiques, les liens constituent un outil puissant de Markdown, et Zettlr les fait passer au niveau supérieur. Zettlr agit de manière très intelligente en ce qui concerne les liens. Rapidement, créons-en un ! Sélectionnez le lien suivant vers notre compte Twitter, et copiez-le dans le presse-papiers : https://www.twitter.com/Zettlr
+Bien qu'ils ne soient pas très souvent utilisés dans les textes académiques, les liens constituent un outil puissant de Markdown, et Zettlr les fait passer au niveau supérieur. Zettlr agit de manière très intelligente en ce qui concerne les liens. Rapidement, créons-en un ! Sélectionnez le lien suivant vers notre compte Twitter, et copiez-le dans le presse-papiers : https://fosstodon.org/@zettlr
```

Comment:
I'm personally in to include the Mastodon account here, but then it should also read "Sélectionnez le lien suivant vers notre compte Mastodon,..." right?

Reply:
Yes! thanks :) : « vers notre compte Mastodon » (sorry)